.github/is-version-number-acceptable.sh

```diff
@@ -12,12 +12,12 @@ then
     exit 0
 fi
 
-current_version=`python setup.py --version`
+current_version=$(python setup.py --version)
 
-if git rev-parse --verify --quiet $current_version
+if git rev-parse --verify --quiet "$current_version"
 then
     echo "Version $current_version already exists in commit:"
-    git --no-pager log -1 $current_version
+    git --no-pager log -1 "$current_version"
```

Comment:
Actually, it happened when I tested super-linter; I got many `shellcheck reported issue in this script: SC2086:info:1:1: Double quote to prevent globbing and word splitting [shellcheck]` errors, hence the quotes :)